CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 9.2.0-alpha.1 - April 21, 2020
+### Features
+ - The [`SymbolLayer.iconTextFit`](https://docs.mapbox.com/android/api/map-sdk/9.1.0/com/mapbox/mapboxsdk/style/layers/PropertyFactory.html#iconTextFit-com.mapbox.mapboxsdk.style.expressions.Expression-) property now respects the stretch metadata of any nine-part stretchable image passed into the `style.addImage` method. You can define the stretchable area of an image by configuring the `stretchX`, `stretchY`, `content` options on `Style.addImage` to append the requisite [metadata](https://github.com/mapbox/mapbox-gl-js/issues/8917). ([#314](https://github.com/mapbox/mapbox-gl-native-android/pull/314))
+ - Added methods to set and get the sort key of features in a `CircleLayer` at runtime. Features with a higher sort key will appear above features with a lower sort key. ([#322](https://github.com/mapbox/mapbox-gl-native-android/pull/322))
```

Comment:
It might be a difference without a distinction, but I think there’s a `PropertyFactory` method to set the sort key now; there isn’t a setter on `CircleLayer` itself.

Reply:
Yeah, good point. Currently this entry is consistent with the changelog entries in 9.1.0 for the other sort key properties we added. I'm OK merging as is, though it can be improved.

To your main point, I am unsure of the best way to cite style properties by name on Android. For the previous entry, I asked @Chaoba whether we should write `PropertyFactory.iconTextFit` or `SymbolLayer.iconTextFit` and he said the latter. I also prefer the latter because it looks better and would be consistent with iOS and GL JS, but I can't tell if `SymbolLayer.iconTextFit` is inaccurate or misleading for developers who use our Android SDK. 

@mapbox/maps-android, could you shed light on how `PropertyFactory` works (e.g. where the relationship between the `PropertyFactory` class and the `*Layer` classes for each layer type is established), and what you would suggest for describing style properties by name?